arch/arm/soc/atmel_sam0/samd/Kconfig.soc

```diff
@@ -0,0 +1,12 @@
+# Kconfig - Atmel SAMD MCU series
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+choice
+	prompt "Atmel SAMD MCU Selection"
+	depends on SOC_SERIES_SAMD
+
+	config SOC_PART_NUMBER_SAMD21G18A
+		bool "SAMD21G18A"
+endchoice
```

Comment:
We should add here all other part numbers which were imported to ext/hal/atmel/asf/sam0/include/samd/

Reply:
Done.